projects/dsp-ui/src/lib/search/advanced-search/advanced-search.component.ts

```diff
@@ -21,9 +21,9 @@ import {
     ResourcePropertyDefinition
 } from '@dasch-swiss/dsp-js';
 import { Subscription } from 'rxjs';
-import { NotificationService } from '../../action';
-import { DspApiConnectionToken } from '../../core';
-import { SearchParams } from '../../viewer';
+import { NotificationService } from '../../action/services/notification.service';
+import { DspApiConnectionToken } from '../../core/core.module';
+import { SearchParams } from '../../viewer/';
```

Comment:
I think this should be `
import { SearchParams } from '../../viewer/views/list-view/list-view.component';`

Reply:
Thanks, somehow missed it. Done in c7971bd.